docs/catalog.rst

```diff
@@ -1,5 +1,41 @@
 .. _sec_catalog:
 
+With ``stdpopsim``, you can run simulations from a number of bespoke demographic models
+that were chosen from notable manuscripts. These models have been rigorously checked 
+and tested by multiple people, so you can rest easy knowing that your simulations are
+reproducible and bug-free!
+
+This catalog shows you all of the possible bespoke simulations that
+you can configure and run with ``stdpopsim``.
+It is organised around a number of choices that you'll need to make:
+
+1. Which **chromosome**? (ie. which :class:`.Genome` object?)
+2. Which **genetic map**? (ie. which :class:`.GeneticMap` object?)
+3. Which **model of demographic history**? (ie. which :class:`.Model` object)
+
+For instance, suppose you are interested in simulating human samples of
+
+1. chromosome 22, using
+2. the HapMapII genetic map, under
+3. a 3-population Out-of-Africa model.
+
+The following command simulates 2 samples from each of the three populations,
+and saves the output to a file called ``test.trees``:
+
+.. code-block:: console
+
+    $ stdpopsim homsap -c chr22 -o test.trees -g HapmapII_GRCh37  --model ooa_3 2 2 2
+
+(To learn more about using ``stdpopsim`` via the command-line, read our tutorial
+about it.)
+
+.. todo::
+	Add a link to the CLI tutorial when it's finished.
+
+Are there other well-known organisms, genetic maps or models that
+you'd like to see in ``stdpopsim``? Head to our :ref:`sec_development`
+page to learn how you can add new items to the catalog.
+
 =======
 Catalog
```

Comment:
This is what I tried doing first -- the issue is that it mucks up the toctree and heading structure of the rest of the Catalog section. In particular, it means that the Species headings no longer appear in the side bar, which I think is quite important for navigability (if that's a word??) I think this is because of the way the `speciescatalog` blocks are configured. There may well be a solution to this that I don't know how to implement -- if anyone with better Rst skills than me wants to take a look, I'd be happy for this to change.

Reply:
Huh. It definitely worked when I just tried it.
<img width="1280" alt="Screen Shot 2019-11-28 at 12 24 09 PM" src="https://user-images.githubusercontent.com/26617086/69824531-0608bc80-11da-11ea-910d-1060e88673f5.png">

I copy pasted the "Catalog" header under the `.. _sec_catalog:`

```
.. _sec_catalog:

=======
Catalog
=======

With ``stdpopsim``, you can run simulations from a number of demographic models
that were implemented from published demographic histories.
```

Try that, and if you can't get it to work, I can make a tiny PR.

Reply:
Hmm, it worked this time! Don't know what was going wrong. The code's been updated.